### PR TITLE
chore: update bug report template to suggest making their Bolt project public

### PR DESCRIPTION
**Pull request by codewithtyler**
The suggested change here is for the person reporting the bug to make their Bolt project public so that it's easier for someone trying to help to see if they can tell what the problem is. By having the project public this would save a bit of back and forth in communication between the maintainer and the person submitting the bug report.

image

### CODE REVIEW

**Review by AriPerkkio — CHANGES_REQUESTED** [review]
Hey @codewithtyler, this change is very welcome!

Maybe we could also have a picture that shows how to do this? I think this picture could be used: https://github.com/stackblitz/bolt.new/issues/3894#issuecomment-2527183627

We could also have a required checkbox with text `I have made my project public or accessible by URL [x]`, like this: https://github.com/vitest-dev/vitest/blob/4e60333dc7235704f96314c34ca510e3901fe61f/.github/ISSUE_TEMPLATE/bug_report.yml?plain=1#L45-L52

**Comment by codewithtyler** [comment]
haha @AriPerkkio I thought you might like this change.

**Comment by codewithtyler** [comment]
Sorry for the multiple commits. GitHub kept pointing me to the wrong line with the file. It kept saying line 2 was the issue when it was really line 33.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -24,6 +24,18 @@ body:
...
+    attributes:
+      label: Validations
+      description: Before submitting the issue, please make sure you do the following
+      options:
+        - label: "Please make your project public or accessible by URL. This will allow anyone trying to help you to easily reproduce the issue and provide assistance."
+          required: true
+  - type: markdown
+    attributes:
+      value: |
+        ![Making your project public](https://utfs.io/f/Z7LZ2jpNjvEALRh1ugzY28iwZcP7IloW1OKsd64SeRGFvMp0)
```
Comment (AriPerkkio) [inline .github/ISSUE_TEMPLATE/bug_report.yml:38]: Let's store this image in the repository at `public/project-visibility.jpg` and reference it directly,.

```suggestion
        ![Making your project public](https://github.com/stackblitz/bolt.new/blob/main/public/project-visibility.jpg?raw=true)
```

Comment (codewithtyler) [reply .github/ISSUE_TEMPLATE/bug_report.yml:38]: Added so now your link should work once it's merged.

**Review by AriPerkkio — APPROVED** [review]
Thanks!

**Pull request merged**